### PR TITLE
Update connect-adfs-wap-report-server.md

### PR DESCRIPTION
**Pull request by robroyle427**
Steps 8 cmdlet isn't the same is the Image above it or below it.  The name could be removed altogether and the cmdlet would return   The picture above has PBIRSWAP.  And the PowerShell command image below has no name specified.  This could be confusing.

### CODE REVIEW

**Comment by PRMerger6** [comment]
@robroyle427 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JoshGer — COMMENTED** [review]
Looked up the Get-WebApplicationProxyApplication cmdlet doc and the requested update is not entirely correct as it needs to include the Name property. So it should be this:

Get-WebApplicationProxyApplication -Name "PBIRSWAP" | FL

I also updated the PowerShell image to reflect the updated command so it matches and sent image to Maggie.

File: powerbi-docs/report-server/connect-adfs-wap-report-server.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ To configure constrained delegation, follow these steps.
 8. Run the following PowerShell command to validate the WAP configuration.
 
     ```
-    Get-WebApplicationProxyApplication "PBIRSBrowser" | FL
+    Get-WebApplicationProxyApplication "PBIRSWAP" | FL
```
Comment (maggiesMSFT) [inline powerbi-docs/report-server/connect-adfs-wap-report-server.md:155]:
```suggestion
    Get-WebApplicationProxyApplication -Name "PBIRSWAP" | FL
```

**Review by maggiesMSFT — CHANGES_REQUESTED** [review]
update

**Comment by PRMerger9** [comment]
@maggiesMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@maggiesMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by maggiesMSFT** [comment]
Thanks for bringing this to our attention, @robroyle427 ! After checking with the product team, it needed to be modified a bit. It also needs an updated image, which I'll do separately. Thanks!

**Pull request merged**